VERSION_INFO

```diff
@@ -1 +1 @@
-0.4.0-dev
\ No newline at end of file
+0.4.1-dev
```

Comment:
The version should not be updated by this PR

Reply:
fixed